test/src/tests/widgets/test_label.py

```diff
@@ -0,0 +1,44 @@
+from pytest import approx, fixture
+
+import toga
+
+from ..test_data import COLORS, TEXTS
+from ..test_utils import set_get
+
+
+@fixture
+async def new_widget():
+    return toga.Label("")
+
+
+async def test_text(widget, probe):
+    for text in TEXTS:
+        set_get(widget, "text", text)
+        assert probe.text == text
+
+
+async def test_color(widget, probe):
+    for color in COLORS:
+        widget.style.color = color
+        for component in ["r", "g", "b"]:
+            assert getattr(probe.color, component) == getattr(color, component)
+        assert probe.color.a == approx(color.a, abs=(1 / 256))
```

Comment:
An example of a property where the value coming out is only approximately equal to the value going in, because of the way it's represented in the native API.

Reply:
That sounds like something we might want to pull out into a utility library; color comparisons are going to be common.

Reply:
Good point, I'll do that.